### PR TITLE
[14.0][FIX/IMP] website_sale_hide_price: Fordward ports from 13.0 to 14.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
- https://github.com/OCA/e-commerce/pull/475
- https://github.com/OCA/e-commerce/pull/639
- https://github.com/OCA/e-commerce/pull/764

@Tecnativa TT41911

@chienandalu @pedrobaeza review please :)

### CODE REVIEW

File: website_sale_hide_price/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class WebsiteSale(WebsiteSale):
+    def _get_products_recently_viewed(self):
+        """When the prices are hidden globally we should force to hide everyone"""
```
Comment (pedrobaeza) [inline website_sale_hide_price/controllers/main.py]:
```suggestion
        """When the prices are hidden globally we should force to hide everywhere."""
```

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — COMMENTED** [review]
@pedrobaeza Do you know why the pre-commit action error?

**Comment by pedrobaeza** [comment]
14.0 branches now require also the copier update.

**Comment by pedrobaeza** [comment]
https://github.com/OCA/e-commerce/pull/768

**Review by chienandalu — APPROVED** [review]
Rebase and let's go to 15.0 :)

**Comment by pedrobaeza** [comment]
Let's not block this for following the chain:

/ocabot merge minor

**Comment by pedrobaeza** [comment]
Ouch, there are pre-commit changes to apply in your code, @pilarvargas-tecnativa

**Comment by pilarvargas-tecnativa** [comment]
Done!

**Comment by chienandalu** [comment]
Ready to merge

**Pull request merged**